sandbox/modules/identity/src/main/java/org/opensearch/identity/extensions/ExtensionsSecretsGenerator.java

```diff
@@ -0,0 +1,22 @@
+/*
+ * SPDX-License-Identifier: Apache-2.0
+ *
+ * The OpenSearch Contributors require contributions made to
+ * this file be licensed under the Apache-2.0 license or a
+ * compatible open source license.
+ */
+
+package org.opensearch.identity.extensions;
+
+import org.apache.commons.lang.RandomStringUtils;
+
+import java.nio.charset.StandardCharsets;
+import java.util.Base64;
+
+public class ExtensionsSecretsGenerator {
```

Comment:
Should we generalize this class to a be  `SecretsGenerator` or maybe add this method to a util class?

Reply:
This could certainly be generalized. I chose to name it `ExtensionsSecretsGenerator` to be clear that methods in here are utilized for extensions.